pom.xml

```diff
@@ -176,7 +176,7 @@ limitations under the License.
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.8.9</version>
+      <version>2.10.1</version>
```

Comment:
This won't work, because jclouds currently does **NOT** support gson >= 2.9.0
See https://issues.jenkins.io/browse/JENKINS-72475 and my corresponding PR #165

Reply:
Thanks for reviewing the PR! I didn't expected that! Just curious how does it sneaked through the tests? Can you give some insights so that I do not repeat the same mistake again.

Reply:
Because there are no tests for that in the plugin. The relevant tests are in jclouds, not in the plugin. It would require connecting to a real cloud provider including the necessary credentials. For obvious reasons, I cannot make the credentials public.

[This](https://issues.jenkins.io/secure/attachment/61904/gson-guice-trace.log) is a stacktrace from before #165 which shows that this happens deep down in jclouds.